api.planx.uk/modules/gis/service/helpers.test.js

```diff
@@ -151,7 +151,6 @@ describe("rollupResultLayer helper function", () => {
     // Assert
     expect(result).toHaveProperty(key);
     layersToRollup.forEach((layer) => {
-      // Jest can handle paths using dot notation, so keys with a dot need to be wrapped in []
```

Comment:
I assume this was supposed to say 'cannot handle...' ?

Reply:
I actually think "can" was correct here - the key here would have been something like `a.b.c` which Jest would have tried to default to a nested path, as opposed to the object in question which is likely `{ "a.b.c": "x" }`

Either way, it's a good call to remove this comment - it looks like vitest [handles this more gracefully](https://vitest.dev/api/expect.html#tohaveproperty) - the test actually passes with or without the wrapping `[]` on line 154.